src/test/java/echo/DeadlineTest.java

```diff
@@ -0,0 +1,31 @@
+package echo;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class DeadlineTest {
+    @Test
+    public void test_creat_deadline() {
+        try {
+            Task deadlineA = new Deadline("dA", "2023/03/23 23:59");
+            Task deadlineB = Task.createTask("deadline dA /by 2023/03/23 23:59");
+            assertEquals(deadlineA.toString(), deadlineB.toString());
+        } catch (DukeException e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    @Test
+    public void test_mark_deadline() {
+        try {
+            Task deadlineA = new Deadline("dA", "2023/03/23 23:59");
+            Task deadlineB = Task.createTask("deadline dA /by 2023/03/23 23:59");
+            deadlineA.setMark();
+            deadlineB.setMark();
+            assertEquals(deadlineA.toString(), deadlineB.toString());
+        } catch (DukeException e) {
+            System.out.println(e.getMessage());
```

Comment:
I think in general it is not adviced to add print statements to tests. I did a quick google search and maybe the JUnit `assertThrows` will be a better way to detect exception and pass/fail the test accoridngly.